app/models/snippet.rb

```diff
@@ -7,13 +7,15 @@
 # Supplejack was created by DigitalNZ at the National Library of NZ
 # and the Department of Internal Affairs. http://digitalnz.org/supplejack
 
+# FIXME: This class is ONLY used by SupplejackCommon
```

Comment:
it Would be a good idea to create these as issues instead of `FIXME:`'s so that it is more visible. (not required)

Reply:
I've added to this story for now:
https://www.pivotaltracker.com/n/projects/152543/stories/154157552